.github/workflows/release.yml

```diff
@@ -0,0 +1,28 @@
+name: Publish Release
+
+on:
+  push:
+    tags:
+      - v*
```

Comment:
The idea with using dynamic versioning via pyproject.toml specifically is to avoid manual creation of version tags, but let them automatically be created from a single source of truth, the `_version.py`.